lnwallet/chainfee/estimator.go

```diff
@@ -406,7 +410,7 @@ func (b *BitcoindEstimator) EstimateFeePerKW(
 //
 // NOTE: This method is part of the Estimator interface.
 func (b *BitcoindEstimator) RelayFeePerKW() SatPerKWeight {
-	return b.minFeePerKW
+	return b.minFeeManager.fetchMinFee()
```

Comment:
@Crypt-iQ re your comment the other day, your statement is _true_ after this commit

Reply:
👍